python/finufftpy/_interfaces.py

```diff
@@ -22,9 +22,16 @@
 from finufftpy.finufftpy_cpp import finufftf_plan
 
 
-### FinufftPlan class definition
-class FinufftPlan:
-    def __init__(self,tp,n_modes_or_dim,iflag,n_trans,eps=None,**kwargs):
+### Plan class definition
+class Plan:
+    def __init__(self,tp,n_modes_or_dim,iflag=None,n_trans=1,eps=None,**kwargs):
+        # set default iflag based on if iflag is None
+        if iflag==None:
+            if tp==2:
+                iflag = -1
+            else:
+                iflag = 1
+
```

Comment:
(This is for below.) Didn't we say that default epsilon should be `1e-6` regardless of precision?

Reply:
I thought we agreed with your suggestion, in the email you said the following,

> * tol. My concern with leaving it at 1e-6 (even for double inputs) is that new users may assume Finufft is simply too inaccurate for their purposes (if they thought 1e-12 was inaccurate enough, imagine how horrified they would be by 1e-6). A safer default might then be to have tolerance being as high as possible for the input datatype (so 1e-6 for single precision, 1e-14 for double precision). We routinely run the 3D code at this precision in Aspire and aren't too concerned about the performance (although maybe we should be?). I think the competition essentially fixes their tolerance at 1e-12 (for whatever that's worth).

I guess I might miss the final discussion for default eps in the Zoom meeting?

Reply:
Right, but I think we discussed it in the meeting and we ended up deciding on a fixed tolerance of `1e-6` for all precision. @ahbarnett can you confirm?

Reply:
I confirm that we decided 1e-6 is default for both precisions. (It's unrealistic for 1d single-prec xforms, but, whatever..)